documentation-website/src/components/header.tsx

```diff
@@ -54,19 +54,24 @@ const Header = ({
       />
     </ul>
   </nav>
-  <nav>
+  <nav aria-label='Main Menu'>
     <ul>
       <li>
         <NavLink to="/quick-start/">
           <Lang lnkey='nav.quick-start'/>
         </NavLink>
       </li>
       <li>
-        <NavLink className="dropdown-links-heading" to="/contributing/">
+        <NavLink
+          aria-expanded="false"
+          id="dropdown-contributors"
+          aria-controls="contributors-menu"
```

Comment:
this is technically the contributing menu - contributors are just one item in it